pkgs/tools/backup/hpe-ltfs/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, fuse, icu, pkgconfig, libxml2, libuuid }:
+
+stdenv.mkDerivation rec {
+  version = "3.4.2_Z7550-02501";
+  pname = "hpe-ltfs";
+
+  src = fetchFromGitHub {
+    rev = version;
+    owner = "nix-community";
+    repo = "hpe-ltfs";
+    sha256 = "193593hsc8nf5dn1fkxhzs1z4fpjh64hdkc8q6n9fgplrpxdlr4s";
+  };
+
+  sourceRoot = "source/ltfs";
+
+  buildInputs = [ 
+    fuse icu pkgconfig libxml2 libuuid
```

```suggestion
    fuse icu  libxml2 libuuid
```

move pkgconfig to `nativeBuildInputs`.